docs/source/index.md

```diff
@@ -1,86 +1,62 @@
 
-```bash
-latch register testworkflow
-```
+<b>Specifying arbitrary cloud compute and storage resources for bioinformatics pipelines is difficult.</b> With Latch SDK, there are several Python task decorators that easily allow you to define the resources available at runtime. The framework starts at 2 CPUs and 4 GBs of memory and goes all the way to 31 CPUs, 120 GBs of memory and 1 GPU (24 GBs of VRAM, 9,216 CUDA cores) to easily handle all processing needs.
 
-This might take 3-10 minutes depending on your network connection. (Subsequent
-registers will complete in seconds by reusing the image layers from this initial
-register.) The registration process will:
+<b>Bioinformatics tools face the challenges of irreproducibility.</b> The lack of proper versioning and dependencies management results in a long tail of poorly documented and unusable bioinformatics software tools.
 
-* Build a docker image containing your workflow code
-* Serialize your code and register it with your LatchBio account
-* Push your docker image to a managed container registry
+Latch SDK containerizes and versions the code in the background each time a workflow is registered to the Latch platform. Container images are constructed by parsing the python runtime for imported libraries or defined using a [Dockerfile](https://docs.docker.com/engine/reference/builder/). Similarly, versions are user-specified as any unique plaintext string. This behavior is a strict requirement of the toolchain and gives us remarkable guarantees with respect to code reproducibility, portability and scalability. 
 
-When registration has completed, you should be able to navigate
-[here](https://console.latch.bio/workflows) and see your new workflow in your
-account.
+## Problem Latch SDK does not yet solve 
+* <b>Workflows chaining</b>: We'll aim to support easy installation and reuse of other Latch SDK workflows in your own workflow.  
+* <b>Workflows monitoring</b>: For batched runs of workflows, we'll aim to provide better dashboard, logs, traces, metrics, and alerting for observability. 
 
-If you are having issues with registration or have general questions, please
-file an issue on [github](https://github.com/latchbio/latch).
+## What Latch SDK is not
+* <b>A pure workflow orchestration engine</b>: There are many popular workflow orchestration engines, such as Nextflow or Snakemake, that can be run locally from a bioinformatician's machine. Although workflow orchestration is a feature of Latch SDK, Latch is fundamentally a cloud-native solution. Latch SDK solves the problems of automating pipelines by providing a robust and scalable cloud infrastructure and asssociated no-code interfaces out of the box. You can also easily bring existing workflow script of any language to Latch (See examples [here](./examples/)).
```

Comment:
"Latch is fundamentally a cloud-native solution." -> "Latch also provides managed infrastructure and no-code interface generation."